cmd/merge-nodes/internal/errors.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import (
+	"errors"
+	"fmt"
+)
+
+var ErrSupervisedNode = errors.New("merging of supervised smeshing nodes is not supported")
+
+type ErrInvalidSchemaVersion struct {
```

Comment:
this type is not needed, in the place where you check it with errors.As you could instead pass the error with expected/actual values and report it in log.Fatal

Reply:
Converted the type to a variable and asserting now with `error.Is`